sklearn/utils/tests/test_utils.py

```diff
@@ -199,11 +202,35 @@ def test_safe_indexing():
     assert_array_equal(np.array(X_inds), np.array(X)[inds])
 
 
+@pytest.mark.parametrize("idx", [0, [0, 1]], ids=['scalar', 'list'])
+def test_safe_indexing_axis_1_sparse(idx):
+    X_true = safe_indexing(X, idx, axis=1)
+
+    X_sparse = sp.csc_matrix(X)
+    assert_array_equal(
```

Comment:
I thought we were moving from these helper?

Reply:
uhm not for the array? We are still using the NumPy testing facility `assert_array_equal`, `assert_allclose` and the dense/sparse version.